subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -152,20 +166,19 @@ def _generate_pod_timings(pod_times, start, end):
         output['container_ready_time'] = (pod_times.containers_ready - pod_times.pod_initialized).total_seconds()
     if hasattr(pod_times, 'containers_ready'):
         output['user_notification'] = (end - pod_times.containers_ready).total_seconds()
-    
+
     return output
 
+
 def _gather_prom_metrics(entry) -> dict:
-    out = {}
-    
-    prom.register()
+    output = {}
 
-    for (key, metric_names) in lts_metrics.items():
+    for cluster_role, metric_names in lts_metrics.items():
```

Comment:
`key` is not descriptive enough, 
here it not a generic dict as above, you know what it contains, so better use a meaningful (and identical everywhere) variable name